form/form_customization.rst

```diff
@@ -735,6 +735,35 @@ class to the ``div`` element around each row:
         </div>
     {% endblock form_row %}
 
+.. tip::
+
+    If you want to customize some instances of the same form only (without
+    the need to create a new form type) you can set the ``block_prefix``
+    option in your form type::
+
+        use Symfony\Component\Form\Extension\Core\Type\TextType;
+        use Symfony\Component\Form\FormBuilderInterface;
+
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder->add('name', TextType::class, array(
+                'block_prefix' => 'wrapped_text',
+            ));
+        }
+
+    .. versionadded:: 4.3
+        The ``block_prefix`` option was introduced in Symfony 4.3.
+
+    Then the block name will be ``wrapped_text_row``.
+
+    .. code-block:: html+twig
+
+        {% block wrapped_text_row %}
+            <div class="wrapped">
+                {{ form_row(form) }}
+            </div>
+        {% endblock wrapped_text_row %}
```

Comment:
in the example above `{% endblock form_row %}` is used, in the examples below only `{% endblock %}`.

What is (should be) the "standard" for twig blocks? cc @weaverryan @javiereguiluz

Reply:
I guess it's related to and depends on the block size, small blocks can end simple `{% endblock %}` and big blocks should end with the block name `{% endblock form_row %}`, but I also think it's a good practice add the block name early, so if your block grows, it's already done.

Reply:
I agree with the first part of Yonel's comment. I don't think this can be automated; it's more a human-based decision. Block name should be added for complex/long blocks, for some (subjective) special cases, when explaining that you can add the block name in the closing tag, etc.